src/front/glsl_new/error.rs

```diff
@@ -12,6 +12,7 @@ pub enum ErrorKind {
     IoError(io::Error),
     ParserFail,
     ParserStackOverflow,
+    NotImplemented(String),
```

Comment:
should this be `&'static str`

Reply:
Heh, I had no luck with `&'a`, but `&'static` should work of course :)